src/test/kotlin/uk/gov/dluhc/printapi/service/BankHolidaysDataServiceTest.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.dluhc.printapi.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.given
+import org.mockito.kotlin.verify
+import uk.gov.dluhc.bankholidaysdataclient.BankHolidayDataClient
+import uk.gov.dluhc.bankholidaysdataclient.BankHolidayDivision
+import uk.gov.dluhc.printapi.testsupport.toRoundedUTCOffsetDateTime
+import java.time.LocalDate
+import java.time.ZoneOffset
+
+@ExtendWith(MockitoExtension::class)
+internal class BankHolidaysDataServiceTest {
```

Comment:
This unit test can obviously move to the shared lib on the assumption you move the impl class `BankHolidaysDataService` there as suggested above ☝️